scripts/pre_commit_linter.py

```diff
@@ -1982,17 +1955,21 @@ def perform_all_lint_checks(self):
             directive_scope_messages + sorted_dependencies_messages +
             controller_dependency_messages +
             html_directive_name_messages + import_order_messages +
-            newline_messages + docstring_messages +
-            comment_messages + html_tag_and_attribute_messages +
+            docstring_messages + comment_messages +
+            html_tag_and_attribute_messages +
             html_linter_messages + linter_messages + pattern_messages +
             copyright_notice_messages)
         return all_messages
 
+
 def _print_complete_summary_of_errors():
     """Print complete summary of errors."""
-    print 'Summary of Errors:'
-    print '----------------------------------------'
-    return _TARGET_STDOUT.getvalue()
+    error_messages = _TARGET_STDOUT.getvalue()
+    if error_messages != '':
```

Comment:
Hmm I am not understanding the use of this, if there are no errors to print then _TARGET_STDOUT.getvalue() will print nothing by default. So why change this?

Reply:
Actually, I have added it, so that the print statements that print the 'Summary of errors' & '---------------',
does not get printed, when we have no errors : As now it also prints that statement, and it seems like there are some errors.